overlord/servicestate/helpers.go

```diff
@@ -97,3 +98,27 @@ func updateSnapstateServices(snapst *snapstate.SnapState, enable, disable []*sna
 	}
 	return true, nil
 }
+
+// MakeServiceQuotaMap builds a map of services and their quota groups. The goal is
+// to make sure that if any quota group applies to a snap service, then we can easily
+// look it up in this based on it's snap.AppInfo. We take the snap instance name, the services
+// want to include in the map (should only be services of the snap name provided) and the quota
+// group of the snap.
+func MakeServiceQuotaMap(snapInfo *snap.Info, grp *quota.Group) map[*snap.AppInfo]*quota.Group {
```

Comment:
Still used in two test packages (wrappers and snapstate/backend), so I can't not export it right? Because I would need to still refer to it from a different package in export_test.go?

Reply:
yea, we need it for tests, it doesn't depend on much though, maybe we can move it to snap/quota ? if we don't tickle circular imports that way

Reply:
It seems like no issue, I'll move it to the snap/quota package and add some unit tests.